Source/CDRRunState.h

```diff
@@ -0,0 +1,11 @@
+#import <Foundation/Foundation.h>
```

Comment:
This file should go into the appraise `Headers` directory to make sure Cocoapods finds it.